modules/ROOT/pages/intro-mule-message.adoc

```diff
@@ -1,9 +1,14 @@
-= Introduction to Mule 4: The Mule Message
+= Introduction to Mule 4: Mule Message
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Mule 4 includes a simplified Mule message model in which each Mule event has a message and variables associated with it. A Mule message is composed of a payload and its attributes (metadata, such as file size). Variables (`vars`) hold arbitrary user information such as operation results, auxiliary values, and so on. This simplified message model makes it easier to work with data in a consistent way across connectors without overwriting information.
+Mule 4 includes a simplified Mule message model in which each Mule event has a message and variables associated with it. A Mule message is composed of: +
```

Comment:
"variables associated with it" ... this means variables are associated with the Mule event or the mule message? If variables are associated with the messgae, I'd suggest rewording to something like: "each Mule event has a message with associated variables".

Reply:
Updated